run_engine.sh

```diff
@@ -663,6 +663,14 @@ then
         git apply tmp.patch
         rm tmp.patch
         cp hadoop-common-project/hadoop-common/src/main/bin/hadoop-functions.sh hadoop-dist/target/hadoop-${version}/libexec/
+    elif [[ $3 == "hdfs_14514" ]]
```

Comment:
This special install command is needed because `install_hdfs.patch` cannot be used for this HDFS version. This is probably needed for all hdfs 2.x versions. We can change the name to `hdfs2` later if this patch is generic for 2.x.